civihr_default_theme/scss/pages/_hr-details.scss

```diff
@@ -65,3 +65,198 @@
     }
   }
 }
+
+.view-mydetails-personal {
+  .contact-id span {
+    @include views-label;
+  }
+
+  .pane-block {
+    border-bottom: 1px solid $gray-light;
+    margin-bottom:0px;
+  }
+
+  .views-row {
```

Comment:
is this for creating the column of content on the right side of the profile picture? If yes, then this selector is way too generic. Can't we have a wrapper element wrapping the whole column, that you then apply the style on?

Reply:
In this case .views-row is the wrapper of the content. I need to add padding to it and a relative position, and then place the profile picture with absolute to the right.